### PR TITLE
turtle.Turtle add the shape keyword.

### PR DESCRIPTION
**Pull request by Dustyposa**
turtle's feature.
A small change for #988 
😄

### CODE REVIEW

**Comment by Dustyposa** [comment]
Other kwargs uncommonly used， so I didn't add the other kwargs.

File: src/lib/turtle.js [hunk 1]
```diff
@@ -565,7 +569,7 @@ function generateTurtleModule(_target) {
             this._down       = true;
             this._color      = "black";
             this._fill       = "black";
-            this._shape      = "classic";
+            // this._shape      = "classic";
```
Comment (albertjan) [inline src/lib/turtle.js]: lets remove this line

Comment (Dustyposa) [reply src/lib/turtle.js]: did it

**Comment by meredydd** [comment]
Reviewed (not run) and accepted!

**Pull request merged**